src/main/config/messages_template.xml

```diff
@@ -438,409 +440,409 @@ See the accompanying LICENSE file for applicable license.
-  
+
   <message id="DOTX005E" type="ERROR">
     <reason>Unable to find navigation title for reference to '%1'.</reason>
-    <response>The build will use '%1' as the title in the Eclipse Table of Contents.</response>
+    <response>Using '%1' as the title in the Eclipse Table of Contents.</response>
   </message>
-  
+
   <message id="DOTX006E" type="ERROR">
-    <reason>Unknown file extension in href="%1".</reason>
-    <response>References to non-DITA resources should set the format attribute to match the resource (for example, 'txt', 'pdf', or 'html').</response>
+    <reason>Unknown file name extension in @href attribute value '%1'.</reason>
+    <response>References to non-DITA resources should set the @format attribute to match the resource (for example, 'txt', 'pdf', or 'html').</response>
   </message>
-  
+
   <message id="DOTX007I" type="INFO">
-    <reason>Only DITA topics, HTML files, and images may be included in your compiled CHM file. The reference to "%1" will be ignored.</reason>
-    <response>To remove this message, you can set the toc="no" or processing-role="resource-only" attribute on your topicref.</response>
-  </message>
-  
-  <!-- As of 20012-06-13, DOTX008W is only called for HTML Help and map2htmtoc -->
-  <message id="DOTX008W" type="WARN">
-    <reason>File '%1' cannot be loaded, and no navigation title is specified for the table of contents.</reason>
-    <response></response>
+    <reason>Only DITA topics, HTML files, and images may be included in the compiled CHM file. Ignoring reference to '%1'.</reason>
+    <response>To remove this message, set the @toc attribute to 'no' or the @processing-role attribute to 'resource-only' on the topic reference.</response>
   </message>
 
   <!-- DOTX008E uses the XSLT prefix DOTX, but generated only from Java code. -->
   <message id="DOTX008E" type="ERROR">
-    <reason>File '%1' does not exist or cannot be loaded.</reason>
+    <reason>The resource '%1' cannot be loaded.</reason>
+    <response>Make sure '%1' exists and that you have permission to access it.</response>
+  </message>
+
+  <!-- As of 2012-06-13, DOTX008W is only called for HTML Help and map2htmtoc -->
+  <message id="DOTX008W" type="WARN">
+    <reason>The resource '%1' cannot be loaded, and no navigation title is specified for the table of contents.</reason>
     <response></response>
   </message>
 
-  <!-- As of 20012-06-13, 009W is only called for HTML Help and map2htmtoc -->
+  <!-- As of 2012-06-13, DOTX009W is only called for HTML Help and map2htmtoc -->
   <message id="DOTX009W" type="WARN">
-    <reason>Could not retrieve a title from '%1'. Using '%2' instead.</reason>
+    <reason>Cannot retrieve a title from '%1'. Using '%2' instead.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX010E" type="ERROR">
-    <reason>Unable to find target for conref="%1".</reason>
-    <response></response>
+    <reason>Unable to find the @conref attribute value '%1'.</reason>
```

Comment:
Technically ... it's not the *value* we cannot find, the value is there in the attribute; we cannot load the target / the thing referenced by that value